lib/src/output_log.rs

```diff
@@ -39,6 +45,13 @@ pub fn create_yaml_file(folder_path: &str, file_name: &str) -> String {
     file_path
 }
 
+pub fn create_log_yaml_file(folder_path: &str, sched_name: &str) -> String {
```

Comment:
It is difficult to tell the difference between this function and the `create_yaml_file` function, so I feel that the process of `create_yaml_file` should be moved inside this function.